package.json

```diff
@@ -19,7 +19,7 @@
     "build-storybook": "build-storybook"
   },
   "dependencies": {
-    "@guardian/src-foundations": "^0.11.0",
+    "@guardian/src-foundations": ">=0.12.4",
```

Comment:
Any reason not to stick with the slightly more conservative `^`? I think `^` will get us to the latest minor version whereas I believe `>=` could end up bumping the major version.

Reply:
yeah agree. I've been burned by not pinning the version in the past :sad_times.

Reply:
I just wanted to ensure we'd **always** get at least this patch, as the previous patch will essentially break our work (by not having the font names we need). I believe the `^` will accept any patch within `0.12`... 🤔 I could be wrong though?

Reply:
Nope, I was wrong - updated the code to `^0.12.4`.